server/admin_management/api/app/repositories/application_admin_repository.py

```diff
@@ -65,3 +66,26 @@ def delete_application_admin(self, application_admin_id: int):
         )
         self.db.delete(record)
         self.db.flush()
+
+    def get_user_app_admin_grants(self, user_id: int) -> List[FamApplication]:
+        """
+        Find out from `app_fam.fam_application_admin` the applications
+            being granted for the user; including "FAM" application.
+
+        Filter on: Only 'IDIR' type user can be an Application Admin.
+
+        :param user_id: primary id that is associated with the user.
+        :return: List of "applications" granted for the user or None.
```

Comment:
shall we say `List of "applications" the user is admin of or None`, just to be more clear?

Reply:
Thanks Catherine for fixing my English sentence :)